main.py

```diff
@@ -1,10 +1,51 @@
 import init_django_orm  # noqa: F401
 
-from db.models import Race, Skill, Player, Guild
+import json
+
+from db.models import Race, Skill, Guild, Player
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        player_data = json.load(f)
+
+    for player_name, player_info in player_data.items():
+        email = player_info["email"]
+        bio = player_info["bio"]
+
+        race_data = player_info["race"]
+        race, created_race = Race.objects.get_or_create(
+            name=race_data["name"],
+            description=race_data["description"]
+        )
+        print(Race.objects.all())
+
+        skills = []
+        for skill_data in race_data.get("skills", []):
+            skill, created_skill = Skill.objects.get_or_create(
+                name=skill_data["name"], race=race, bonus=skill_data["bonus"]
+            )
+            skills.append(skill)
+        print(Skill.objects.all())
```

Comment:
Simplify Skills creation
There is no need to store skill into variable.
There is no need to append it to list